components/BookCard.js

```diff
@@ -1,21 +1,22 @@
 import styled from 'styled-components';
 
-export default function Card({ title, author }) {
+export default function Card({ title, author, onDelete }) {
```

Comment:
Hey Doro, looking great!
The only minor remark I have is that you might want to rename the function to BookCard so it matches the name of your file at some point but that's all. :)